browser/net/brave_site_hacks_network_delegate_helper_unittest.cc

```diff
@@ -105,7 +106,8 @@ TEST(BraveSiteHacksNetworkDelegateHelperTest, OnionReferrerStripped) {
   const GURL original_referrer(
       "https://"
       "brave4u7jddbv7cyviptqjc7jusxh72uik7zt6adtckl5f4nwy2v72qd.onion/");
-  const GURL destination("https://brave.com");
+  const GURL destination =
+      url::Origin::Create(GURL("https://brave.com")).GetURL();
```

Comment:
what's wrong with `const GURL destination("https://brave.com");`?

Reply:
Sorry, I think I over-did this one :sweat_smile:, fixed in https://github.com/brave/brave-core/commit/02855182fa0d19d311ec82c3ac847310c2899106

Reply:
That might have been me on the account of url1->origin->url2 doesn't mean url1 == url2, but yeah, obviously, in this case it isn't a concern.